vector/src/main/java/im/vector/app/features/call/webrtc/WebRtcCall.kt

```diff
@@ -269,17 +271,28 @@ class WebRtcCall(
     private fun createPeerConnection(turnServerResponse: TurnServerResponse?) {
         val peerConnectionFactory = peerConnectionFactoryProvider.get() ?: return
         val iceServers = mutableListOf<PeerConnection.IceServer>().apply {
-            turnServerResponse?.let { server ->
-                server.uris?.forEach { uri ->
+            if (vectorPreferences.useFallbackTurnServer()) {
+                listOf("turn:turn.matrix.org:3478?transport=udp", "turn:turn.matrix.org:3478?transport=tcp", "turns:turn.matrix.org:443?transport=tcp").forEach {
```

Comment:
Maybe extract this list to the file https://github.com/vector-im/element-android/blob/main/vector/src/main/res/values/config.xml#L2 ?